### ISSUE TITLE
Why do you multiply param values by 1 in REBNCONV?

### ISSUE THREAD

**Issue by advaza** [NathanUA/U-2-Net#69]
From the model spec in `u2net.py`

```python
class REBNCONV(nn.Module):
    def __init__(self,in_ch=3,out_ch=3,dirate=1):
        super(REBNCONV,self).__init__()

        self.conv_s1 = nn.Conv2d(in_ch,out_ch,3,padding=1*dirate,dilation=1*dirate)
        self.bn_s1 = nn.BatchNorm2d(out_ch)
        self.relu_s1 = nn.ReLU(inplace=True)

    def forward(self,x):

        hx = x
        xout = self.relu_s1(self.bn_s1(self.conv_s1(hx)))

        return xout
```

Why do you multiply by 1 the parameters:  `padding=1*dirate,dilation=1*dirate` ?

Thanks
Adva

**Comment by NathanUA** [comment]
Insightful point. This is just for the convenience in the model development stage. You can ignore that now.
